portal/models/intervention_strategies.py

```diff
@@ -439,9 +439,17 @@ def completed_card_html(assessment_status):
             # If the user was enrolled in indefinite work and lands
             # here, they should see the thank you text.
             if assessment_status.enrolled_in_classification('indefinite'):
-                card_html = thank_you_block(
-                    name=user.display_name,
-                    registry=assessment_status.organization.name)
+                logout_label = _("Log Out")
+                card_html = """
+                            {thank_you}
+                            <div class="button-container portal-header-logout-container">
+                                <a class="btn-lg btn-tnth-primary" href="/logout">{logout}</a>
+                            </div>
+                            """.format(
+                                      thank_you=thank_you_block(name=user.display_name,
```

Comment:
I'm unclear about why this case of "thank you" would be different from the other.  Shouldn't we simply update the `thank_you_block` function to include the logout button?